src/argilla/client/sdk/v1/datasets/models.py

```diff
@@ -112,7 +112,7 @@ class FeedbackQuestionModel(BaseModel):
 class FeedbackMetadataPropertyModel(BaseModel):
     id: UUID
     name: str
-    description: Optional[str] = None
+    title: Optional[str] = None
     visible_for_annotators: Optional[bool] = True
```

Comment:
Something similar for this one, it should always have a value:

```suggestion
    visible_for_annotators: bool
```